src/helpers/to-id.ts

```diff
@@ -12,3 +12,8 @@ export const toNoteId = (value: string) => {
 export const toDailyNoteId = (date: Date) => {
   return format(date, 'ddMMyyyy')
 }
+
+export const isDailyNoteId = (value: string) => {
+  const match = value.match(/^(\d{2})(\d{2})(\d{4})$/)
+  return !!match
```

Comment:
```suggestion
  return (/^(\d{2})(\d{2})(\d{4})$/).test(value)
```

Reply:
But I want to make sure it returns boolean

Reply:
ah I see now